src/components/Nav/Nav.js

```diff
@@ -1,5 +1,62 @@
 import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Search from './components/Search/Search';
+import * as S from './Nav.style';
 
 export default function Nav() {
-  return <div>Nav</div>;
+  const userToken = localStorage.getItem('UserToken');
+  const navigate = useNavigate();
+
+  const toPosting = () => {
+    if (userToken === null) {
+      alert('로그인이 필요한 서비스입니다. \n로그인하시겠습니까?');
+      navigate('/login');
+    } else if (userToken !== null) {
```

Comment:
```suggestion
    } else if (userToken) {
```
null이 아니라는건 있다는거니까 
userToken이라고 해도 됩니다.